fli-iam/shanoir-ng
# [fli-iam#2312] Correct stats proc acquisition import_date

# Changes

Fix stats export following renaming of field  `dataset_acquisition.creation_date` to `dataset_acquisition.import_date` (see https://github.com/fli-iam/shanoir-ng/pull/2283)

# How to test
- Execute `docker-compose/database/3_add_statistics_procedure.sql` on OFSEP qualif
- Download statistics
- Check SQL change script numbering before merge
